bokehjs/src/lib/models/sources/column_data_source.ts

```diff
@@ -10,21 +10,37 @@ import * as typed_array from "core/util/typed_array"
 import {keys} from "core/util/object"
 
 //exported for testing
-export function stream_to_column(col: Arrayable, new_col: Arrayable, rollover?: number): Arrayable {
+// FIXME : Too complex
+export function stream_to_column(col: Arrayable, new_col: Arrayable, append: boolean, rollover?: number): Arrayable {
```

Comment:
@stark71  The JS tests are currently failing and I think it is because this signature breaks existing usage. However, I think this is possible:
```
function stream_to_column(col: Arrayable, new_col: Arrayable, rollover?: number, append = True)
```
and if so I think this is the appropriate change as it leaves the current use of the API undisturbed.

Reply:
FYI here are the failing JS unit tests output: https://travis-ci.org/bokeh/bokeh/jobs/407864562#L3148

Reply:
As an FYI the reason the tests are failing I assume is because some of our tests are still written in CoffeeScript, so there is no error when those tests call your new `stream` *without* providing a value for `append`. If we want to make `append` mandatory another option is to fix up the tests to pass a value for `append` always. 

also cc @mattpap please comment if you have thoughts. My guess

Reply:
I used your suggestion.